backend/api/auth_register.go

```diff
@@ -50,6 +50,11 @@ func (server *Server) RegisterUser(ctx *gin.Context) {
 		return
 	}
 
+	if err = util.NotifyUser(user.PhoneNumber, "Thank you for registering for GoodGrocer! You will recieve updates about your requests and errands through this number as you use the app!"); err != nil {
```

Comment:
Catching the error here might be problematic because you could create the user but not send a successful response back.

Reply:
ah yeah good point, forgot to take that out

Reply:
but also same argument for the token generation on line 59, should i take that out too?

Reply:
yeah but token generation failure is extremely rare + token generation is very important for the registration flow